### DISCUSSION TITLE
I cant get plain FTP to work

### DISCUSSION THREAD

**Discussion by adnseek** [drakkan/sftpgo#1363] [Q&A]
Hi,
i'm trying to use port 2121 for plain FTP, and everytime the client connects i see this 

![image](https://github.com/drakkan/sftpgo/assets/120753123/f7339afa-3edb-4fd2-ba6e-4830b43fb222)

while sftp works great on port 2022.

Does anyone have an idea what i might do wrong here ?

Greets
Adrian

**Comment by captainhook** [comment]
Did you manage to resolve this?
